### DISCUSSION TITLE
Save wizard feature

### DISCUSSION THREAD

**Discussion by RedKetchum1527** [bucanero/apollo-ps4#111] [Ideas]
Pls add a feature which has the sama save wizard feature like decrypting or encrypting save  , in next update we can decrypt an encrypted save and export decrypted files back to it .

**Comment by bucanero** [comment]
Apollo can already decrypt data files from ps4 save-games, and import+encrypt them back to the original save.

**Comment by RedKetchum1527** [comment]
But why am I getting error after putting it back
* I copy it from hdd to usb and click on "Copy save game"
" Unplug usb and plug it in pc then open save editor of rdr2 , edit it like
changing horses and changing outfits after that saving those changes in it
, I unplug usb and plug it back to ps4
* Open Apollo ps4 save tool and copy it back to hdd without messing with
other options like import or export only copy save game back to ps4

**Reply by bucanero** [reply]
The save-game can have additional security checks, like checksums and hashes. If those checksums are not fixed by the save-editor, then the game will detect the save as invalid.

I don't know the editor or tools you're using, perhaps they're meant to be used with Save Wizard to rebuild the checksums. I suggest to ask on Discord servers related to save editing, they might have more information.
